tripal_chado/src/Task/ChadoPreparer.php

```diff
@@ -158,6 +191,624 @@ public function performTask() :bool {
     return $task_success;
   }
 
+
+
+    /**
+   * Many of the custom tables created for Chado v1.2 are now in Chado v1.3.
+   *
+   * These tables need not be tracked by Tripal anymore as custom tables and
+   * in some cases the Chado version has different columns so we need to
+   * adjust them.
+   */
+  protected function fix_v1_3_custom_tables() {
+    
+    $connection_chado = \Drupal::service('tripal_chado.database');
```

Comment:
Is there a reason we're not using `$this->chado` here as set on line 132?

Reply:
Good catch. I'll fix it.